### PR TITLE
Create introducing-files-v2-1.svx

### PR DESCRIPTION
**Pull request by yaichenbaum**
This contains the changelog for Files v2.1 and should be merged when the release goes live.

### CODE REVIEW

**Review by TheOnlyTails — APPROVED** [review]

File: src/routes/blog/posts/introducing-files-v2-1.svx
Comment (chingucoding) [inline src/routes/blog/posts/introducing-files-v2-1.svx:13]: nit: Maybe we could rephrase this a bit to make it more clear that this was already present? First time I read this, I was confused because the blog is about 2.1 and not 2.0.33. But maybe this is just me here.

**Review by chingucoding — APPROVED** [review]

File: src/routes/blog/posts/introducing-files-v2-1.svx
Comment (chingucoding) [reply src/routes/blog/posts/introducing-files-v2-1.svx]: The new formulation is great, feel free to merge this @yaichenbaum if you are ready.

Comment (yaichenbaum) [reply src/routes/blog/posts/introducing-files-v2-1.svx]: I still have a couple screenshots to upload and to finalize the release.

File: src/routes/blog/posts/introducing-files-v2-1.svx
Comment (yaichenbaum) [inline src/routes/blog/posts/introducing-files-v2-1.svx:4]: @TheOnlyTails is the dash preventing the thumbnail from working?

Comment (yaichenbaum) [reply src/routes/blog/posts/introducing-files-v2-1.svx:4]: Whoops, the image didn't finish uploading.

Comment (yaichenbaum) [reply src/routes/blog/posts/introducing-files-v2-1.svx:4]: Whoops, the image didn't finish uploading.

**Comment by TheOnlyTails** [comment]
Is this ready?

**Comment by yaichenbaum** [comment]
> Is this ready?

@TheOnlyTails just waiting on the store release to become available

**Pull request merged**
